docs/spelling.txt

```diff
@@ -43,7 +43,9 @@ decrypt
 decrypted
 doesn
 embeddable
+freetype
```

Comment:
You can just add the keywords at the top of the package.rst file. Don't edit this one.